### PR TITLE
Add tiller section to the cluster installation documentation

### PR DESCRIPTION
**Pull request by jakkab**
Confirm these statements before you submit your pull request:

- [x] I have read and submitted the required [Contributor Licence Agreements](https://github.com/kyma-project/community/blob/master/CONTRIBUTING.md#agreements-and-licenses).
- [x] This pull request follows the contributing guidelines, recommended Git workflow, and templates.
- [x] I have tested my changes.
- [x] I have updated the relevant documentation.
---

**Description**

Changes proposed in this pull request:

- Add tiller section to the cluster installation documentation

### CODE REVIEW

File: installation/README.md [hunk 1]
```diff
@@ -77,15 +77,37 @@ When you fill in all required placeholder values, run the following command to p
 kubectl apply -f installer-config.yaml
 ```
 
-3. Deploy the `Installer` component.
+3. Bind the default RBAC role
```
Comment (klaudiagrz) [inline installation/README.md]: Missing dot at the end of the sentence.

File: installation/README.md [hunk 2]
```diff
@@ -77,15 +77,37 @@ When you fill in all required placeholder values, run the following command to p
...
+
+Kyma inallation requires increased permissions granted by the **cluster-admin** role. To bind the role with the default **ServiceAccount**, run the following command: 
+
+```
+kubectl apply -f resources/cluster-prerequisites/default-sa-rbac-role.yaml
+```
+
+4. Deploy `tiller`
```
Comment (klaudiagrz) [inline installation/README.md]: Missing dot

File: installation/README.md [hunk 3]
```diff
@@ -77,15 +77,37 @@ When you fill in all required placeholder values, run the following command to p
...
+
+To deploy the `tiller` component on your cluster, run the following command:
+
+```
+kubectl apply -f installation/resources/tiller.yaml
+```
+
+Wait for the `tiller` pod to be ready. To make sure it is running, execute the following command:
```
Comment (klaudiagrz) [inline installation/README.md]: Pod

**Comment by jakkab** [comment]
@klaudiagrz Comments addressed!

**Pull request merged**